packages/flame/lib/src/camera/camera_component.dart

```diff
@@ -271,4 +271,10 @@ class CameraComponent extends Component {
       boundedBehavior.bounds = bounds;
     }
   }
+
+  /// Returns true if the given [component] is currently visible through this
+  /// camera.
+  bool inView(PositionComponent component) {
```

Comment:
I think Dart's naming convention would be to call this method `isInView`: https://dart.dev/guides/language/effective-dart/design#prefer-a-non-imperative-verb-phrase-for-a-boolean-property-or-variable

Reply:
Changed it!

Reply:
Oops -- sorry, my bad... Now it sounds like the camera is in view of a component: `if (camera.isInView(component)) ...`. Perhaps `hasInView` would sound better?

Reply:
Hmm, what about `sees`? `if (camera.sees(component)) ...`

Reply:
Yeah, that's even better

Reply:
Changed it!